src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -6,7 +6,7 @@
 public class RhymersJUnitTest {
 	DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
 	FirstInFirstOutRhymer FIFORhymer = new FirstInFirstOutRhymer();
-	final int EMPTY_STACK_VALUE = -1;
+	final int EMPTY_STACK_VALUE = 0;

```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).